CMakeLists.txt

```diff
@@ -776,6 +776,7 @@ if(WIN32)
     add_definitions(-DFEATURE_COMINTEROP_APARTMENT_SUPPORT)
     add_definitions(-DFEATURE_COMINTEROP_UNMANAGED_ACTIVATION)
     add_definitions(-DFEATURE_COMINTEROP_WINRT_MANAGED_ACTIVATION)
+    add_definitions(-DFEATURE_DELAYLOAD_WINRT)
```

Comment:
This may be a silly question but why don't we just use the linker option for delay loading?